README.md

```diff
@@ -105,36 +105,78 @@ This theme uses the following files for linting:
 
 ### Syncing Environments
 
-The preferred mechanism for syncing your environment with others is to use database imports and exports. This repo has a few scripts to make this process as easy as possible. While your containers are running, you can run any of these commands to import, export, or backup a database. Here are the quick commands, with more instructions below.
+The preferred mechanism for syncing your environment with others is to use database/uploads imports and exports. This repo has a few scripts to make this process as easy as possible. While your containers are running, you can run any of these commands to import, export, or backup a database or your `uploads` folder. Here are the quick commands, with more instructions below.
 
 ```sh
-# import a DB from the `sql` folder
-npm run import-db
+# import a DB from the `sync/sql` folder
+npm run import:db
+
+# import images/files into your uploads folder from a zip file in `sync/uploads`
+npm run import:uploads
+
+# run both a DB import and an uploads import at once
+npm run import
 
 # export your DB
-npm run export-db
+npm run export:db
 
 # export your DB with a custom name
-npm run export-db validation-data
+npm run export:db validation-data
+
+# export your uploads folder as a zip file
+npm run export:uploads
+
+# export your uploads with a custom name
+npm run export:uploads validation-uploads
+
+# run both a DB export and an uploads export at once
+npm run export
+
+# run DB/uploads exports with a custom name
+npm run export validation
 
 # backup your DB in case you need to restore it later
-npm run backup-db
+npm run backup:db
 
 # backup your DB with a custom name
-npm run backup-db work-in-progress
+npm run backup:db work-in-progress-data
+
+# backup your uploads folder in case you need to restore it later
+npm run backup:uploads
+
+# backup your uploads folder with a custom name
+npm run backup:uploads work-in-progress-uploads
+
+# run both a DB backup and an uploads backup at once
+npm run backup
+
+# run DB/uploads backups with a custom name
+npm run export work-in-progress
 ```
 
 #### Importing Databases
 
-You can import databases from production, a saved backup, or another developer's DB export with the `import-db` script. To use it, put a `*.sql.gz` file in a top-level `sql` folder in the repo and run `npm run import-db`. This will first back up your existing database in case you need to revert back to it, and then it will import the database from the given file, effectively replacing your database with a new one.
+You can import databases from production, a saved backup, or another developer's DB export with the `import:db` script. To use it, put a `*.sql.gz` file in a top-level `sync/sql` folder in the repo and run `npm run import:db`. This will first back up your existing database in case you need to revert back to it, and then it will import the database from the given file, effectively replacing your database with a new one.
```

Comment:
```suggestion
You can import databases from production, a saved backup, or another developer's DB export with the `import:db` script. To use it, put a `*.sql.gz` file in directly inside of a `sync/sql` folder in the repo and run `npm run import:db`. This will first back up your existing database to `sync/sql/backups` in case you need to revert back to it, and then it will import the database from the given file, effectively replacing your database with a new one. The imported sql script with then be moved to `sync/sql/previous-imports`.
```